lib/inquisition/auditors/backend/rails_best_practices/installer.rb

```diff
@@ -0,0 +1,19 @@
+module Inquisition
+  module Auditors
+    module Backend
+      module RailsBestPractices
+        class Installer < Auditors::Installer
+          class << self
+            private
+
+            def configers
```

Comment:
```suggestion
            def configurers
```
again, no such word exists